tests/unit/core/server/testServer.js

```diff
@@ -31,13 +32,22 @@ describe('core/server/base', () => {
 
   function testClient({ stubRoutes = defaultStubRoutes } = {}) {
     function createStoreAndSagas() {
+      const sagaMiddleware = createSagaMiddleware();
       return {
-        store: createStore(combineReducers({ reduxAsyncConnect })),
-        sagaMiddleware: null,
+        store: createStore(
+          combineReducers({ reduxAsyncConnect }),
+          undefined,
```

Comment:
Not my fault :) `createStore` is just an [awkward API](http://redux.js.org/docs/api/createStore.html). Passing `undefined` seems like the least intrusive way to skip the argument (I do not want to define the second argument).